### PR TITLE
DOCS: Cover scrapy-bench in the documentation

### LINKED ISSUE #4996: Cover scrapy-bench in the documentation
Cover https://github.com/scrapy/scrapy-bench in our [documentation topic about benchmarking](https://docs.scrapy.org/en/latest/topics/benchmarking.html#benchmarking), as a project to use for more complex Scrapy benchmarking.

It may also be a good idea to remove the promise about future `scrapy bench` enhancements.

### PR DESCRIPTION
**Pull request by deepang17**
Resolves #4996.

### CODE REVIEW

File: docs/topics/benchmarking.rst [hunk 1]
```diff
@@ -81,5 +81,5 @@ follow links, any custom spider you write will probably do more stuff which
 results in slower crawl rates. How slower depends on how much your spider does
 and how well it's written.
 
-In the future, more cases will be added to the benchmarking suite to cover
-other common scenarios.
+To use it as a project for more complex Scrapy benchmarking:
+https://github.com/scrapy/scrapy-bench
```
Comment (Gallaecio) [inline docs/topics/benchmarking.rst:85]: What about a shorter “Use scrapy-bench for more complex benchmarking”? (linking “scrapy-bench” to https://github.com/scrapy/scrapy-bench)

**Comment by deepang17** [comment]
@Gallaecio Please review the changes. Thank you

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```
Comment (Gallaecio) [inline .vscode/settings.json:3]: Please, remove this file.

See https://docs.github.com/es/github/using-git/ignoring-files, you might want to set up a global rule in your system to exclude those files, unless you are working on a project that keeps track of those.

File: docs/topics/benchmarking.rst [hunk 3]
```diff
@@ -81,5 +81,6 @@ follow links, any custom spider you write will probably do more stuff which
 results in slower crawl rates. How slower depends on how much your spider does
 and how well it's written.
 
-In the future, more cases will be added to the benchmarking suite to cover
-other common scenarios.
+.. _scrapy-bench: https://github.com/scrapy/scrapy-bench
+
+Use scrapy-bench_ for more complex benchmarking.
```
Comment (Gallaecio) [inline docs/topics/benchmarking.rst:86]: :lipstick: Usually, links are placed at the end of the section or page.
```suggestion
Use scrapy-bench_ for more complex benchmarking.

.. _scrapy-bench: https://github.com/scrapy/scrapy-bench
```

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**